app/assets/stylesheets/modules/masthead.css

```
@@ -84,6 +84,10 @@
     --bs-btn-bg: var(--stanford-digital-red);
     --bs-btn-border-color: var(--stanford-digital-red);
   }
+
+  button#search {
+    padding: 0.625rem 1.25rem;
```

can we set the variables here? 
```suggestion
    --bs-btn-padding-y: 1.25rem;
    --bs-btn-padding-x: 0.625rem;
```

---

done